docs/pages/workflow/expo-cli.md

```diff
@@ -237,17 +257,87 @@ This command will be disabled if your project is configured to use `metro` for b
 '$ npx expo prebuild',
 ]} cmdCopy="npx expo prebuild" />
 
-Native source code must be generated before a native app can compile. Expo CLI provides a unique and powerful system called _prebuild_, that generates the native code for your project.
+Native source code must be generated before a native app can compile. Expo CLI provides a unique and powerful system called _prebuild_, that generates the native code for your project. To learn more, read the [Expo Prebuild docs](/workflow/prebuild.md).
 
-**For more information:**
+## Config
 
-- [Prebuild](/workflow/prebuild)
-- [Config plugins](/guides/config-plugins)
-- [Native modules API](/modules/module-api)
+Evaluate the Expo config (**app.json**, or **app.config.js**) by running:
 
-## Telemetry
+<Terminal cmd={[
+'$ npx expo config',
+]} cmdCopy="npx expo config" />
+
+- `--full`: Include all project config data
```

Comment:
```suggestion
- `--full`: Include all project config data.
```

The other two bullet points are ending with a period. For consistency, I'm adding one here.